app/static/scss/admin/navbar.scss

```diff
@@ -113,8 +149,57 @@ $w-9: 9rem;
     font-size: 2.5em;
 }
 
+@media screen and (max-width: $phone-max) {
+    nav {
+        left: (-1 * $nav-wide) !important;
+        width: $nav-wide;
+
+        .nav-item {
+            width: $nav-wide !important;
+        }
+
+        &.open {
+            left: 0 !important;
+            .ham {
+
+                .image {
```

Comment:
I think that the class "image" here doesn't really describe what this button does.  What is "image?"